sqlmesh/utils/info.py

```diff
@@ -0,0 +1,71 @@
+from typing import Any, Optional, Set
```

Comment:
Let's call make this module more descriptive by calling it `config.py` since everything here is related to configuration.

Reply:
Alright! I updated the PR.